src/app/modals/role/role.component.pug

```diff
@@ -0,0 +1,16 @@
+.modal.fade-in.show
+  .modal-dialog(role="document")
+    .modal-content
+      .modal-header
+        h4.modal-title([innerHtml]="'Complete your Profile' | translate")
+        button.close(type="button", (click)="dismissModal()", aria-label="Close")
+          span(aria-hidden="true") &times;
+      .modal-body.btn-group-vertical
+        p([innerHtml]="'You&apos;re a...' | translate")
+        a.btn.btn-secondary((click)="saveRole('ROLE_CU_UNDERGRAD_STUDENT')") {{ 'Undergraduate Student' | translate }}
+        a.btn.my-sm-1.btn-secondary((click)="saveRole('ROLE_CU_GRAD_STUDENT')") {{ 'Graduate Student' | translate }}
+        a.btn.my-sm-1.btn-secondary((click)="saveRole('ROLE_CU_FACULTY')") {{ 'Instructor/Faculty' | translate }}
+        a.btn.my-sm-1.btn-secondary((click)="saveRole('ROLE_PS_TEACHER')") {{ 'K-12 Teacher' | translate }}
+        a.btn.my-sm-1.btn-secondary((click)="saveRole('ROLE_CU_LIBRARIAN')") {{ 'Librarian' | translate }}
+        a.btn.my-sm-1.btn-secondary((click)="saveRole('ROLE_OTHER')") {{ 'Other' | translate }}
+        a.link.my-sm-1.btn-link((click)="dismissModal()") {{ 'No thanks' | translate }}
```

Comment:
I don't know artstor very well but are we able to get the same UX with these underlying elements being buttons. They feel more like buttons rather than links that would perform some sort of navigation.

Reply:
@michael-iden only the last element, the no thanks link, is a link. Everything else is a button!

Reply:
I'm not super familiar with pug but I'm reading this as these elements are all links (anchor HTML tags) but styled to look like buttons in some cases. From an accessibility standpoint, I think we've gotten dinged on HTML elements that are styled to look like other HTML elements since sometimes they read out differently with screen readers and may have slightly different keyboard controls.

Reply:
![image](https://user-images.githubusercontent.com/13315416/135670318-06e7e401-624b-4ead-9f35-ea4c51e80d56.png)

Reply:
I think you're right, thanks for clarifying. I'm not super familiar with pug either.